segment/assets/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Get started with your Twilio Functions!</title>
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+    <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+    <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+    <style>
+      body {
+        padding: 20px;
+        display: flex;
+        flex-direction: column;
+        min-height: 100vh;
+      }
+
+      div[role="main"] {
+        flex: 1;
+      }
+
+      footer {
+        text-align: center;
+      }
+
+      footer p {
+        margin-bottom: 0;
+      }
+
+      #twilio-logo {
+        width: 50px;
+        height: 50px;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to your new Twilio App</h1>
+    </header>
+    <div role="main">
+      <section>
+        <h3>Get started with your app</h3>
+        <p>This app sends an SMS to your number when your Segment-enabled website generates a specific tracking event.</p>
+        <ol>
+          <li>
+            Add a new webhook destination for your website's Segment Connection, with <code><span class="function-root"></span>/track-sms</code> as the URL.
+          </li>
+          <li>
+            Wait for your Segment-enabled website to generate <code>track</code> events with the name you configured for this function.
```

Comment:
Is there a way in Segment to manually trigger a specific event?

Reply:
There doesn't seem to be anything built into Segment to do this, but I can ask our Segment contact if there's any way to do it programmatically.

Reply:
The Segment event tester allows this functionality; the up-to-date `index.html` contains a pointer towards it in the Troubleshooting section, alongside a screenshot.